cmd/uhc/config/cmd.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright (c) 2019 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package configcmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift-online/uhc-cli/cmd/uhc/config/get"
+	"github.com/openshift-online/uhc-cli/cmd/uhc/config/set"
+)
+
+var args struct {
+	debug bool
+}
+
+var Cmd = &cobra.Command{
+	Use:   "config COMMAND VARIABLE",
+	Short: "get or set about configuration variables",
+	Run:   run,
```

Comment:
Not completely sure, but I think that for this intermediate command we don't need the `run` function at all. Can you check and if so remove it, please?